bindings/node/test/tools/utils.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = Object.create(null);
```

Comment:
We already have a file that builds a "requirements" object for tests to use (`requirements.helpers.js`).  Can we move this code there?

Also, following the pattern set in that file, it might be nice to simply export a boolean indicating whether or not the credential provider is installed (considering we'll be adding support for gcp / azure soon as well).

Maybe exporting something like this:
```javascript
{
  credentialProvidersInstalled: { 
    aws: <boolean>
  }
}
```